### DISCUSSION TITLE
Relationship to formal  Shark IQ integration?

### DISCUSSION THREAD

**Discussion by short4bmoney** [sharkiqlibs/sharkiq#26] [Q&A] [answered]
I stumbled on this repo after looking at the older one by @ajmarks and I saw signs that pointed to control for newer models, including support for features like room selection and mapping.

I have the [formal integration](https://www.home-assistant.io/integrations/sharkiq) installed but the `send_command` service doesn't seem to work for my model RV2500.

Should I expect the progress on this repo to ultimately make its way to the formal integration or are folks side-loading this integration via HACS or another way to take advantage of newer features?

**Comment by funkybunch** [comment, marked as answer]
Hey @short4bmoney,

This is the repo for the official core SharkIQ integration in home assistant, as the other repo had been stale with no response from the single-repo owner for over a year (we have 3 maintainers of this repo at the moment to prevent that from happening again).

The service you're talking about is supported by the underlying Python library (this repo), but the method has not been integrated into Home Assistant as a service yet.  If you have bandwidth to submit a PR on the main Home Assistant Core repo then I'd be happy to expedite a review.

I've had an unconventionally busy summer so I won't have a chance to look at making my own PR for another couple days.  I'd love to have that and the auth token refresh issue implemented for the next major release of Home Assistant.

Hope this helps!

**Reply by funkybunch** [reply]
Also, the formal integration does not support the `send_command` service for any robots.  Again, we'll expedite reviews for PRs submitted to the Home Assistant repo.  This library should have the underlying functions needed to create a `send_command` service.
